factory/glideFactoryMonitorAggregator.py

```diff
@@ -280,11 +182,12 @@ def aggregateStatus(in_downtime):
     for entry in monitorAggregatorConfig.entries:
         # load entry status file
         status_fname = os.path.join(
-            os.path.join(monitorAggregatorConfig.monitor_dir, "entry_" + entry), monitorAggregatorConfig.status_relname
+            monitorAggregatorConfig.monitor_dir, "entry_" + entry, monitorAggregatorConfig.status_relname
         )
         # load entry completed data file
         completed_data_fname = os.path.join(
-            os.path.join(monitorAggregatorConfig.monitor_dir, "entry_" + entry),
+            monitorAggregatorConfig.monitor_dir,
+            "entry_" + entry,
```

Comment:
Would it be better to change `"entry_" + entry` to `f"entry_{entry}"`?

Reply:
done